### PR TITLE
Fixed lightGallery displays wrong product image after changing variation.

### PR DESCRIPTION
**Pull request by evemf**
…change.

### Ticket
- [Frontend: Fehler Bild / Bildunterschrift bei variablen Produkten](https://app.asana.com/0/30156186242982/1127101697339500/f)

### CODE REVIEW

File: assets/scripts/script.js [hunk 1]
```diff
@@ -62,5 +62,18 @@
...
+     * Modifies data-srcset of product gallery first image on variation change.
+     *
+     * lightGallery gets the srcset from the anchor element that wraps the
+     * image. On variation change, WooCommerce changes the srcset of the first
+     * product image with the variation one, but not the srcset of the wrapping
+     * anchor element.
+     */
+    $('.variations_form').on('woocommerce_variation_has_changed', function() {
+      const firstImage = $('.woocommerce-product-gallery__image img').eq(0);
+      firstImage.parent().attr('data-srcset', firstImage.attr('srcset'));
```
Comment (sun) [inline assets/scripts/script.js]: Should we modify the parent if it's not the expected?
```suggestion
      firstImage.parent('[data-srcset]').attr('data-srcset', firstImage.attr('srcset'));
```
Or even account for potential other markup structures?
```suggestion
      firstImage.parents('[data-srcset]').attr('data-srcset', firstImage.attr('srcset'));
```
Also, is there always a first image?
```suggestion
      if (firstImage.length) {
        firstImage.parents('[data-srcset]').attr('data-srcset', firstImage.attr('srcset'));
      }
```

**Comment by sun** [comment]
Please squash-merge using the corrected PR title as commit message.

**Comment by LucaPipolo** [comment]
@evemf — please also remember to double check plugin version in all places (including `README` file).

**Comment by sun** [comment]
@evemf Please delete the feature branch after merging a PR (using the button directly here in the GitHub UI). Make sure that it actually refers to the feature branch though; sometimes it suggests to delete the entire repository.

**Comment by sun** [comment]
@evemf Luca was referring to the "Stable Tag" in the readme.txt:
https://github.com/netzstrategen/wordpress-gallerya/blob/ecc38054f0961ca3464b717523d151a5592f320f/README.txt#L6

Can you add another commit directly to master to update the version number, please?

**Comment by evemf** [comment]
@sun I haven't been able to do a commit into master (well, the push is what I am not allowed to). so, I created a branch and opened a PR just for it :S
https://github.com/netzstrategen/wordpress-gallerya/pull/29

**Pull request merged**